test-lit/fir/fir-ops.fir

```diff
@@ -0,0 +1,341 @@
+// Test the FIR operations
```

Comment:
Maybe a comment for our future selves -- Is this a round-trip test?  Should all new fir operations be added to this test?  Is it complete right now?

Reply:
Yes. Yes. I believe so, but was going to check one more time.

Reply:
Cool.  I was asking more for a comment with a potential TODO section.

Reply:
I haven't written a (Tilikum) bridge test. That's TODO, yes.